bedrock/mozorg/templatetags/misc.py

```diff
@@ -599,6 +599,51 @@ def ifeq(a, b, text):
     return jinja2.Markup(text if a == b else '')
 
 
+@library.global_function
+@jinja2.contextfunction
+def app_store_url(ctx, base_url):
+    """Returns a localised app store URL for a given product"""
+    locale = getattr(ctx['request'], 'locale', 'en-US')
+    countries = settings.APPLE_APPSTORE_COUNTRY_MAP
+    if locale in countries:
+        return base_url.format(country=countries[locale])
+    else:
+        return base_url.replace('/{country}/', '/')
+
+
+@library.global_function
+@jinja2.contextfunction
+def play_store_url(ctx, base_url):
+    """Returns a localised play store URL for a given product"""
+    locale = getattr(ctx['request'], 'locale', 'en-US')
+    return base_url + '&hl=' + locale.split('-')[0]
```

Comment:
Could be simplified by using your helper function below, removing the line above, and an F string.
```suggestion
    return f'{base_url}&hl={lang_short(ctx)}'
```